servermon/hwdoc/migrations/0002_auto_20160614_1333.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('hwdoc', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='project',
+            name='comments',
+            field=models.TextField(null=True, blank=True),
```

Comment:
You thought correctly. What you say holds true for all fields except text related fields (CharField, TextField, etc). In text related fields the empty value is '' and not NULL like it would be for an IntegerField.